### PR TITLE
Add section on tone and content

### PR DESCRIPTION
**Pull request by gtewallace**
Suggestion from https://github.com/nodejs/nodejs-collection/pull/2/commits/de65c3d8cad90903c5c0aec4b78c61c7874ae73e

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
 # nodejs-collection
 A place for the Node.js Collective to organize the editing process
 
+## Tone and Content
+
+We are looking for posts that inform, teach, experiment, and generally enrich our community. There's no hard and fast rule of what an article should or feel like; Node.js Collection articles should be as diverse as the community itself. Ideally, contributions should encourage readers to explore new ideas, dig deeper into old ones, and to inspire readers to experiment and tinker. Articles can cover any number of topics such as:
```
Comment (Trott) [inline README.md]: typo: `an article should or feel like` 

(Also: .md files in our repos tend to get wrapped at 80 chars, but I'm certainly OK if this repo doesn't want to do that. Just pointing it out if you want to conform for the sake of uniformity.)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
+
+If you're passionate about something not listed above, please don't hesitate to submit your idea for consideration.
```
Comment (Trott) [inline README.md]: Nit: `passionate` is a bit of a cliché and is a term that's been under a fair bit of scrutiny and criticism lately for other reasons.  I personally don't mind it, but others might. Consider perhaps using `excited` instead?

**Comment by bnb** [comment]
I want to make sure @ZibbyKeaton is aware of this, since she's the one kicked off the creation of the original document 😊

**Comment by ZibbyKeaton** [comment]
I think these changes are totally fine. Please feel free to update word choices, no problem with me.

**Comment by gtewallace** [comment]
thanks for getting on it so fast @bnb and @Trott 

+1 from me on all the recommended edits.

**Comment by Trott** [comment]
This content still seems A-OK to me, but a rebase is needed so it can be inserted into the right place with all the other content that's been added since this was opened. I'm going to close this for now, but if anyone wants to re-open it and keep working on it, by all means, re-open (if GitHub allows) or leave a comment here.

**Pull request closed without merging**